src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,103 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.RegistrationUserException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int DEFAULT_AGE = 18;
+    private static final int LOW_AGE = 16;
+    private static final int NEGATIVE_AGE = -1;
+    private static final String DEFAULT_PASS = "password";
+    private static final String SHORT_PASS = "short";
```

Comment:
Consider using a more descriptive constant name for the short password, such as 'SHORT_PASSWORD'.